test/mitmproxy/contentviews/test_urlencoded.py

```diff
@@ -12,4 +12,7 @@ def test_view_urlencoded():
     d = url.encode([("adsfa", "")]).encode()
     assert v(d)
 
+    d = url.encode([]).encode()
+    assert v(d)
```

Comment:
I think it would be much more useful to write a test that directly invokes `base.format_dict` in contentviews/test_base.py 😃

Reply:
Why didn't I think of that, will do it ASAP